### PR TITLE
update docs

### PR DESCRIPTION
**Pull request by serverglen**

### CODE REVIEW

**Comment by serverglen** [comment]
Thread 1 (Thread 0x7f75d1cf19c0 (LWP 28604)):
#0  0x00007f75d1477783 in tcmalloc::ThreadCache::ReleaseToCentralCache(tcmalloc::ThreadCache::FreeList*, unsigned long, int) () from /usr/lib/libtcmalloc_and_profiler.so.4
#1  0x00007f75d147783b in tcmalloc::ThreadCache::ListTooLong(tcmalloc::ThreadCache::FreeList*, unsigned long) () from /usr/lib/libtcmalloc_and_profiler.so.4
#2  0x00007f75d1489ee8 in tc_deletearray () from /usr/lib/libtcmalloc_and_profiler.so.4
#3  0x0000000000422900 in testing::internal::UnitTestImpl::~UnitTestImpl() ()
#4  0x00000000004220d3 in testing::UnitTest::~UnitTest() ()
#5  0x00007f75cf8f1ff8 in ?? () from /lib/x86_64-linux-gnu/libc.so.6
#6  0x00007f75cf8f2045 in exit () from /lib/x86_64-linux-gnu/libc.so.6
#7  0x00007f75cf8d8837 in __libc_start_main () from /lib/x86_64-linux-gnu/libc.so.6
#8  0x0000000000406639 in _start ()
[runtest] 'bthread_unittest' failed, 18 succeeded

bthread ut 出core，本次PR只是修改几个文档，应该和当前PR没有关系。

**Comment by zyearn** [comment]
restarted

**Comment by serverglen** [comment]
> restarted

应该是单测不稳定。

File: docs/cn/bvar_c++.md [hunk 1]
```diff
@@ -148,7 +148,7 @@ int expose_as(const butil::StringPiece& prefix, const butil::StringPiece& name);
 ```
 全局曝光后的bvar名字便为name或prefix + name，可通过以_exposed为后缀的static函数查询。比如Variable::describe_exposed(name)会返回名为name的bvar的描述。
 
-当相同名字的bvar已存在时，expose会打印FATAL日志并返回-1。如果选项**--bvar_abort_on_same_name**设为true (默认是false)，程序会直接abort。
+当相同名字的bvar已存在时，expose会打印FATAL日志并返回-1。如果选项 **-bvar_abort_on_same_name**设为true (默认是false)，程序会直接abort。
```
Comment (lorinlee) [inline docs/cn/bvar_c++.md:151]: 看起来风格是**加重的两遍都加空格，这里"设为"前面也加一下空格吗

Comment (serverglen) [reply docs/cn/bvar_c++.md:151]: 设为前面不用加。

File: docs/cn/overview.md [hunk 2]
```diff
@@ -66,7 +66,7 @@ RPC不是万能的抽象，否则我们也不需要TCP/IP这一层了。但是
 * 访问服务? 包含[brpc/channel.h](https://github.com/brpc/brpc/blob/master/src/brpc/channel.h)并参考注释或[示例](https://github.com/brpc/brpc/blob/master/example/echo_c++/client.cpp).
 * 调整参数? 看看[brpc/controller.h](https://github.com/brpc/brpc/blob/master/src/brpc/controller.h). 注意这个类是Server和Channel共用的，分成了三段，分别标记为Client-side, Server-side和Both-side methods。
 
-我们尝试让事情变得更加简单，以命名服务为例，在其他RPC实现中，你也许需要复制一长段晦涩的代码才可使用，而在brpc中访问BNS可以这么写"bns://node-name"，DNS是`Init("http://domain-name", ...)`，本地文件列表是"file:///home/work/server.list"，相信不用解释，你也能明白这些代表什么。
+我们尝试让事情变得更加简单，以命名服务为例，在其他RPC实现中，你也许需要复制一长段晦涩的代码才可使用，而在brpc中访问BNS可以这么写"bns://node-name"，DNS是`"http://domain-name"`，本地文件列表是"file:///home/work/server.list"，相信不用解释，你也能明白这些代表什么。
```
Comment (lorinlee) [inline docs/cn/overview.md:69]: 这里BNS的"bns://node-name"没有加\`\`，http加了，为啥不统一呀

Comment (serverglen) [reply docs/cn/overview.md]: 已修改

**Pull request merged**